src/Uno.Toolkit.UI/Controls/TabBar/TabBarSelectionIndicatorPresenter.cs

```diff
@@ -344,7 +390,7 @@ private void UpdateSelectionIndicatorPosition(Point? destination = null)
 				return;
 			}
 
-			StopStoryboards();
+			StopRunningAnimation();
 
 			templateSettings.IndicatorTransitionFrom = templateSettings.IndicatorTransitionTo;
```

Comment:
whats the point of updating IndicatorTransitionFrom in StopRunningAnimation if you are going to override them immediately here?
or did you mean to update IndicatorTransitionTo instead? (`TemplateSettings.IndicatorTransitionTo= TemplateSettings.IndicatorTransitionTo with { XY = ...}`)